lib/utils/utils.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class Strings {
+  static const username = 'Username';
+  static const password = 'Password';
+  static const no_accont = "Don't have an account? ";
+  static const sign_up = "Sign up";
+  static const forgot_psswd = 'Forgot your password?';
+  static const login = 'LOGIN';
+  static const login_fb = 'Sign up with Facebook';
```

Comment:
signUp instead of sign_up, please keep this convention for all variables.

Reply:
Got ur point thanx

Reply:
also, you have a type in accont. It should be account. Also, don't use abbreviations: password instead of passwd, facebook instead of fb